src/test/java/com/epam/steps/admin/CreateTeacherSteps.java

```diff
@@ -30,24 +32,29 @@ public void fillInNameSurname() {
 
     @Then("User is created and displayed in the list")
     public void userIsCreatedAndDisplayedInTheList() {
-        assertThat(adminPage.checkNewUserIsDisplayedOnAdminsSection()).isTrue();
+        assertThat(adminPage.checkNewUserIsDisplayedOnAdminsSection())
+                .withFailMessage("Last created user is not displayed in the list, but it should be.")
+                .isTrue();
     }
 
     @Then("Check teacher is not added in the DB")
     public void checkAdminIsNotAddedInTheDB() {
```

Comment:
Please, change the method name to "checkTeacherIsNotAddedInTheDB()"

Reply:
done